sources/tools/Stride.Graphics.RenderDocPlugin/Stride.Graphics.RenderDocPlugin.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
   <Import Project="..\..\targets\Stride.props" />
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I remember our build system used to have issues with `TargetFramework` but it's likely not the case anymore (maybe that was when we used RIDs for graphics API).
Anyway, if it works well, all good!